content/blog/understanding-sql-dialects.md

```diff
@@ -0,0 +1,65 @@
++++
+title = "Understanding SQL Dialects: How Different Databases Speak Their Own SQL Language"
+slug = "demo-post"
+date = 2024-06-26T11:19:22+05:30
+image = "/images/2024/understanding_sql_dialects.png"
+draft = true
+authors = ["Syed Mohd Mehndi"]
+description = "This blog post aims to provide a general overview of how different DBMSs extend standard SQL with their unique features, making it easier for developers to understand and utilize these systems effectively."
+tags = ["SQL", "Database", "RDBMS", "Postgres", "Oracle", "SQL Server"]
+categories = ["SQL", "Database"]
```

Comment:
- Lets update the slug to something like `understanding-sql-dialects`
- Before merging, draft needs to be set as `false`
- @swathynair19 can you please suggest some existing tags/categories that this blog can be linked to

Reply:
We can keep the tags short to just two:  "SQL" and "DBMS".

Reply:
Thanks @2KAbhishek  resolved point number 1,2 and 3
@swathynair19 updated the file to use only "SQL" and "RDBMS"